src/renderer/services/chain/memo.ts

```diff
@@ -0,0 +1,34 @@
+import { getDepositMemo } from '@thorchain/asgardex-util'
+import * as FP from 'fp-ts/lib/function'
+import * as O from 'fp-ts/lib/Option'
+import * as Rx from 'rxjs'
+import * as RxOp from 'rxjs/operators'
+
+import { BASE_CHAIN } from '../../const'
+import { eqChain } from '../../helpers/fp/eq'
+import { sequenceTOption } from '../../helpers/fpHelpers'
+import { selectedPoolAsset$ } from '../midgard/common'
+import { baseAddress$ } from './address'
+import { MemoRx } from './types'
+
+/**
+ * Stake memo
+ */
+const stakeMemo$: MemoRx = Rx.combineLatest([selectedPoolAsset$, baseAddress$]).pipe(
+  RxOp.switchMap(([oPoolAsset, oBaseAddress]) =>
+    Rx.of(
```

Comment:
do we actually need this ? 
why not just `RxOp.map` instead of `RxOp.switchMap` ?